R/dettl_runner.R

```diff
@@ -1,11 +1,13 @@
 #' Create an import object using functions defined at specified path
 #'
 #' @param path Path to directory containing functions for import process.
+#' @param db_name The name of the db to connect to. Connection info must be
+#' configured via the `db_config.yml`.
 #'
 #' @return A dataImport object.
 #' @export
 #'
-new_import <- function(path) {
+new_import <- function(path, db_name) {
```

Comment:
in `orderly` we default to the first listed name in the configuration (so take a `NULL` argument here). What do you think?

Reply:
Also, and in a different ticket probably, `new_import` is an odd name - it sounds like something you might run to create the skeleton of a new import (we have this in `orderly` as `orderly::orderly_new`). Not sure what a better name would be though

Reply:
Using a default is a good idea, I'll do that. 

Yeah noted the point about the `new_import` I was probably thinking about what the method was doing instead of thinking from user perspective. They're initialising an or setting up an import for a directory and a database so `setup_import` or `initialise_import` is probably more appropriate. I can add a separate ticket for that as you suggested.